### ISSUE TITLE
Exception when I use make singlehtml

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#474]
Hi all,

I wanted to update the documentation of the flask-testing extension, but I have got this error with the make singlehtml command.

```
#!python
stephane@stargate ~/p/f/docs> make singlehtml
sphinx-build -b singlehtml -d _build/doctrees   . _build/singlehtml
Running Sphinx v1.0
loading pickled environment... done
building [singlehtml]: all documents
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
preparing documents... done
assembling single document... 
writing... 
Exception occurred:
  File "/usr/local/lib/python2.6/dist-packages/Sphinx-1.0-py2.6.egg/sphinx/builders/html.py", line 835, in fix_refuris
    for refnode in tree.traverse(nodes.reference):
AttributeError: 'NoneType' object has no attribute 'traverse'
The full traceback has been saved in /tmp/sphinx-err-mKGXDI.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
Either send bugs to the mailing list at <http://groups.google.com/group/sphinx-dev/>,
or report them in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
make: *** [singlehtml] Error 1
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/474
- Originally reported by: Stéphane Wirtel
- Originally created at: 2010-07-27T10:59:55.943

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-07-27 11:46:02+00:00_

Fixed in <<cset 8c0c728d1bee97e3c286784db062067175a24269>>. Thanks!

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-22 10:14:42+00:00_

Removing milestone: 1.0 (automated comment)
